app/jitsi/main.py

```diff
@@ -13,19 +13,21 @@ def join():
 
 @main.route('/rooms')
 def get_rooms():
-    basedir = current_app.config.get('BASE_DIR')
     rooms = current_app.config['ROOMS']
     adventures = current_app.config['ADVENTURES']
     for adventure in adventures.values():
+        config = adventure.get('config', {})
```

Comment:
this way of setting up rooms/adventures still feels pretty brittle to me, but i don't claim to have a better solution on hand...

Reply:
I agree -- this `config` stuff is, in my mind, temporary. I am ok with the way rooms are set up but I definitely want to reform the adventures. I made a ticket for this yesterday: https://github.com/morganecf/jitsi-party/issues/128 

Not entirely sure what the solution is but I at least want to think about it some more